unity/Assets/Scripts/AgentManager.cs

```diff
@@ -1355,69 +1355,121 @@ public class ObjectMetadata {
+    // back to uncooked state with contextual interactions, only a direct action
+    public bool cookable;
+
+    // is it cooked right now? - context sensitive objects might set this
+    // automatically like Toaster/Microwave/Pots/Pans if isHeated = true
+    // temperature placeholder values, might get more specific later
+    // with degrees but for now just track these three states
+    public bool isCooked;
+
+    // return current abstracted temperature of object as a string (RoomTemp, Hot, Cold)
+    public string temperature;
+
+    // can change other object temp to hot
+    public bool isHeatSource;
+
+    // can change other object temp to cool
+    public bool isColdSource;
+
+    // can this be sliced in some way?
+    public bool sliceable;
+
+    // currently sliced?
+    public bool isSliced;
+
+    // can this object be opened?
     public bool openable;
+
+    // is this object currently opened?
     public bool isOpen;
-    public float openness; // if the object is openable, what is the current openness? It's a normalized percentage from [0:1]
-    ///
+
+    // if the object is openable, what is the current openness? It's a normalized value from [0:1]
+    public float openness;
+
+    // can this object be picked up?
     public bool pickupable;
-    public bool isPickedUp;// if the pickupable object is actively being held by the agent
-    public bool moveable;// if the object is moveable, able to be pushed/affected by physics but is too big to pick up
 
-    public float mass;// mass is only for moveable and pickupable objects
+    // if the pickupable object is actively being held by the agent
+    public bool isPickedUp;
+
+    // if the object is moveable, able to be pushed/affected by physics but is too big to pick up
+    public bool moveable;
+
+    // mass is only for moveable and pickupable objects
+    public float mass;
 
-    // salient materials are only for pickupable and moveable objects, for now static only objects do not report material back since we have to assign them manually
-    public enum ObjectSalientMaterial { Metal, Wood, Plastic, Glass, Ceramic, Stone, Fabric, Rubber, Food, Paper, Wax, Soap, Sponge, Organic, Leather } // salient materials that make up an object (ie: cell phone - metal, glass)
+    // Salient materials that this object is made of as strings (see enum above).
+    // This is only for objects that are Pickupable or Moveable
+    public string[] salientMaterials;
 
-    public string[] salientMaterials; // salient materials that this object is made of as strings (see enum above). This is only for objects that are Pickupable or Moveable
-    ///
     public string[] receptacleObjectIds;
-    public float distance;// dintance fromm object's transform to agent transform
-    public String objectType;
+
+    // distance from object's transform to agent transform
+    public float distance;
+
+    // what type of object is this?
+    public string objectType;
```

Comment:
Also note: I've changed `String` to `string`. There didn't seem to be any reason the `String` class was being used here.

Most of the other changes just move the comments above the fields so the lines are super long.